packages/@navikt/prosessmeny/src/step.tsx

```diff
@@ -0,0 +1,89 @@
+import bem from '@navikt/bem-utils';
+import classnames from 'classnames';
+import { Normaltekst } from 'nav-frontend-typografi';
+import * as React from 'react';
+import './stepStyles';
+
+/* eslint-disable global-require */
+const advarselImgPath = require('./assets/images/advarsel.svg') as string;
+const avslaatImgPath = require('./assets/images/avslaatt_valgt.svg') as string;
+const checkImgPath = require('./assets/images/check.svg') as string;
+/* eslint-enable global-require */
+
+export enum StepType {
+    warning = 'warning',
+    success = 'success',
+    danger = 'danger',
+    default = 'default',
+}
+
+interface StepProps {
+    label: string;
+    index: number;
+    isFinished?: boolean;
+    isActive?: boolean;
+    isDisabled?: boolean;
+    type?: StepType;
+    onClick?: (index: number) => void;
+}
+
+const stepCls = bem('step');
+
+export const Step = React.memo(
+    ({ label, index, isActive, onClick, isDisabled, isFinished, type = StepType.default }: StepProps): JSX.Element => {
+        const handleButtonClick = (event: React.FormEvent<HTMLButtonElement>): void => {
+            event.preventDefault();
+            onClick(index);
+        };
+
+        const getIcon = (): JSX.Element => {
+            if (isFinished) {
+                if (type === StepType.warning) {
+                    return (
+                        <img
+                            src={advarselImgPath}
+                            alt="Behandlet - Manuell oppgave"
+                            className={stepCls.element('icon', 'warning')}
+                        />
+                    );
+                }
+                if (type === StepType.danger) {
+                    return (
+                        <img
+                            src={avslaatImgPath}
+                            alt="Oppgave løst/avslått"
+                            className={stepCls.element('icon', 'danger')}
+                        />
+                    );
+                }
+                return (
+                    <img
+                        src={checkImgPath}
+                        alt="Behandlet - Oppgave løst/godkjent"
+                        className={stepCls.element('icon', 'success')}
+                    />
+                );
+            }
+            return <span className={stepCls.element('icon-placeholder')} />;
+        };
+        const stepIndicatorCls = classnames(stepCls.element('indicator', type), {
+            [stepCls.element('indicator', 'active')]: isActive,
```

Comment:
Note to self (dette tror jeg er koden jeg selv skrev):
Vi må lage en egen funksjon i bemUtil for elementWithModifier. `.element(x,y)` med 2 argumenter blir forvirrende.